src/components/apphost.js

```diff
@@ -1,456 +1,441 @@
+    if (browser.tizen) {
+        deviceName = 'Samsung Smart TV';
+    } else if (browser.web0s) {
+        deviceName = 'LG Smart TV';
+    } else if (browser.operaTv) {
+        deviceName = 'Opera TV';
+    } else if (browser.xboxOne) {
+        deviceName = 'Xbox One';
+    } else if (browser.ps4) {
+        deviceName = 'Sony PS4';
+    } else if (browser.chrome) {
+        deviceName = 'Chrome';
+    } else if (browser.edge) {
+        deviceName = 'Edge';
+    } else if (browser.firefox) {
+        deviceName = 'Firefox';
+    } else if (browser.opera) {
+        deviceName = 'Opera';
+    } else if (browser.safari) {
+        deviceName = 'Safari';
+    } else {
+        deviceName = 'Web Browser';
+    }
 
-        return deviceName;
+    if (browser.ipad) {
+        deviceName += ' iPad';
+    } else if (browser.iphone) {
+        deviceName += ' iPhone';
+    } else if (browser.android) {
+        deviceName += ' Android';
     }
 
-    function supportsVoiceInput() {
-        if (!browser.tv) {
-            return window.SpeechRecognition || window.webkitSpeechRecognition || window.mozSpeechRecognition || window.oSpeechRecognition || window.msSpeechRecognition;
-        }
+    return deviceName;
+}
 
-        return false;
+function supportsVoiceInput() {
+    if (!browser.tv) {
+        return window.SpeechRecognition || window.webkitSpeechRecognition || window.mozSpeechRecognition || window.oSpeechRecognition || window.msSpeechRecognition;
     }
 
-    function supportsFullscreen() {
-        if (browser.tv) {
-            return false;
-        }
+    return false;
+}
 
-        var element = document.documentElement;
-        return (element.requestFullscreen || element.mozRequestFullScreen || element.webkitRequestFullscreen || element.msRequestFullscreen) || document.createElement('video').webkitEnterFullscreen;
+function supportsFullscreen() {
+    if (browser.tv) {
+        return false;
     }
 
-    function getSyncProfile() {
-        return new Promise(function (resolve) {
-            require(['browserdeviceprofile', 'appSettings'], function (profileBuilder, appSettings) {
-                var profile;
-
-                if (window.NativeShell) {
-                    profile = window.NativeShell.AppHost.getSyncProfile(profileBuilder, appSettings);
-                } else {
-                    profile = profileBuilder();
-                    profile.MaxStaticMusicBitrate = appSettings.maxStaticMusicBitrate();
-                }
+    const element = document.documentElement;
+    return (element.requestFullscreen || element.mozRequestFullScreen || element.webkitRequestFullscreen || element.msRequestFullscreen) || document.createElement('video').webkitEnterFullscreen;
+}
 
-                resolve(profile);
-            });
-        });
-    }
+function getSyncProfile() {
+    return new Promise(function (resolve) {
+        require(['browserdeviceprofile', 'appSettings'], function (profileBuilder, appSettings) {
```

Comment:
``` suggestion
Promise.all([
    import('browserdeviceprofile'),
    import('appSettings')
])
.then(([profileBuilder,appSettings]) => {
```

I think that's the right way to handle multiple imports

Reply:
Still needs to change?

Reply:
I removed getSyncProfile, since we don't have sync (so it's effectively useless code)